bundles/framework/layerlist/view/LayerViewTabs/LayerList/LayerCollapse/LayerCollapseHandler.js

```diff
@@ -15,7 +15,9 @@ const getLayerGroups = (groups = []) => {
     return groups.map(group => {
         return {
             id: group.id,
-            name: Oskari.getLocalized(group.name),
+            name: group.name,
+            description: group.description,
```

Comment:
Use getters here instead of internal variables:
```suggestion
            name: group.getTitle(),
            description: group.getDescription(),
```